README.md

```diff
@@ -1,2 +1,17 @@
-# Kobiton Samples
-This repository includes examples to demonstrate how to execute web/application Automation testing using [Kobiton](https://kobiton.com/) (**real devices** mobile cloud platform) in different programming languages.
+# Run automation test on Kobiton devices using Katalon tool
```

Comment:
cc. @thaitrongnghi 

This README file is for all samples, should keep it as before. My suggestion was to put everything on this PR under a separate folder, below is my suggested structure

```
|- README <-- this file
|- Java
|- JavaScript <-- existing folders
|- Frameworks
       |- robot <-- this is for Nghi
       |- Katalon
               |- README
               |- ...
```